consensus/istanbul/backend/engine.go

```diff
@@ -559,6 +564,20 @@ func (sb *Backend) Seal(chain consensus.ChainReader, block *types.Block, results
 		return err
 	}
 
+	parentExtras, err := types.ExtractIstanbulExtra(parent)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("Got seal from extras: ", parentExtras.Bitmap, parentExtras.CommittedSeal)
+	block = block.WithParentSeal(
```

Comment:
I think the block passed to `Seal` should already have the `parentSeal` populated